libp2p/P2PMessage.h

```diff
@@ -106,5 +106,22 @@ enum AMOPPacketType
     RequestTopics = 2,
     SendTopics = 3
 };
+enum TopicStatus
+{
+    INIT_STATUS = 0,
```

Comment:
please give some comment for these statuses.

Reply:
done